R/ncbi_snp_api.R

```diff
@@ -211,7 +220,7 @@ get_gene_names <- function(primary_info) {
 #' ncbi_snp_query("rs121909001")
 #' ncbi_snp_query("rs121909001", verbose = TRUE)
 #' }
-ncbi_snp_query <- function(snps) {
+ncbi_snp_query <- function(snps, version = "38") {
   
```

Comment:
Could you add a `@param` describing version and then regenerate the docs? This will fix the failing github builds.